.circleci/orbs/helix-post-deploy/orb.yml

```diff
@@ -158,7 +158,7 @@ commands:
                 actionVersion=`node -e "console.log(require('./package.json').version.match(/^[0-9]+/)[0])"`
                 nrURL="--url https://adobeioruntime.net/api/v1/web/${actionNS}/${actionPackage}/${actionName}@v${actionVersion}/_status_check/healthcheck.json"
               fi
-              if [ $aws = true ]; then
+              if [ "$aws" = true ]; then
```

Comment:
As mentioned in another review, this condition causes a runtime error if `aws` is not defined at all.